testing/btest/Baseline/language.record-duplicate-fields/.stderr

```diff
@@ -1,3 +1,2 @@
 ### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
-error in <...>/record-duplicate-fields.zeek, line 7: Duplicate field 'a' found in record definition
-
+error in <...>/record-duplicate-fields.zeek, line 7: duplicate field 'a' found in record definition
```

Comment:
This test (record-duplicate-fields) wasn't crashing on master. Would you be up for adding a new one for the scenario you found? Seems the following does it:
```
$ cat language/record-duplicate-fields-global.zeek 
# @TEST-EXEC-FAIL: zeek -b %INPUT
# @TEST-EXEC: TEST_DIFF_CANONIFIER=$SCRIPTS/diff-remove-abspath btest-diff .stderr

type test: record {
        a: count;
        a: count;
};

global a: test;
```